### DISCUSSION TITLE
推理不支持batch的推理

### DISCUSSION THREAD

**Discussion by qrsssh** [RapidAI/RapidLatexOCR#19] [Q&A] [answered]
您好， 非常感谢您的开源，看您的代码只支持单图的推理，是否考虑batch的批量推理代码的开发

**Comment by qrsssh** [comment]
@SWHL  多谢多谢

**Reply by SWHL** [reply]
最近没时间搞，欢迎贡献代码哈

**Comment by SWHL** [comment, marked as answer]
刚才认真看了你的需求，并做了评估。
大概思路是：从推理引擎层面考虑，支持batch的推理，需要将多张图做拼接，转为相同shape，才可以batch推理。
但是这样会存在识别结果不准的问题。不同大小的图像，被统一为同样大小一起识别，这势必会对某些图像造成影响。

建议：可以多开几个并发来实现
